source/features/comments-time-machine-links.tsx

```diff
@@ -9,7 +9,7 @@ function init() {
 
```

Comment:
Popped up a discussion about this fix for those interested, https://twitter.com/nickytonline/status/1117524648982392833

Reply:
It is actually, I saw it on WHATWG I think, it’s be worth updating dom.d.ts as well if you want

Reply:
Yeah, going to open an issue probably and PR it.